### PR TITLE
New Assignment Page: make input fields consistent

### LINKED ISSUE #555: Unequal width and length of Input Fields
**To Reproduce**
Can be found on http://localhost:8080/groups/1/assignments/new

**Screenshots**
Issue

Also, unequal width of input fields for "Name", "Grading Scale" and "Deadline"

### PR DESCRIPTION
**Pull request by yummypasta**
#### Describe changes made in this PR
I changed the width and height of the `Name` and `Deadline` input fields in the New Assignment page to visually match the styling of the `Grading scale` input field.

### Screenshots
Screen Shot 2020-01-01 at 9 40 11 PM

### CODE REVIEW

**Comment by yummypasta** [comment]
I actually could not test this after merging `CircuitVerse/master` into my branch. With the new UI, none of the navbar buttons work, I could not go to the sign in page, and I could not view my groups page.

Is there a way I can test this? It theoretically should work, but I don't know if my CSS changes translate over to SCSS 100%.

**Comment by Ananthavijay** [comment]
@yummypasta  make the changes locally and test it out.. Don't include the navigation problem in this PR as it has already been added as an beginner task

**Comment by aayush-05** [comment]
@yummypasta use writing urls in the search bar to get to a certain page, the route links are all same. For example, use localhost:3000/users/sign_in to get into the login window section and move forward.
Or you can try gitpod as an easy way, but I am not sure if the login works in that.

**Comment by yummypasta** [comment]
@aayush-05 I've tried using direct URLs, but they either redirect (such as `/users/sign_in`) back to the home page, or they send the error:
```
The record you wish access could not be found
```
Is there a workaround?

**Comment by aayush-05** [comment]
It is working fine for me
image

Please check that you are running the latest commit from the master. Also check the comment I made on the GCI dashboard

**Comment by yummypasta** [comment]
@aayush-05 Sorry, I just had leftover cookies and I didn't realize that the "login" button gets replaced with "notifications." I've updated my changes to be consistent with #623. I've updated the screenshots on the original comment.

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -13,6 +13,11 @@
   height: 25px;
 }
 
+#assignment_name, #party {
```
[the remark this answers is not in the document]
Comment (JoshHeng) [reply app/assets/stylesheets/layout.scss]: Either use inline styling or a class instead

[the remark this answers is not in the document]
Comment (JoshHeng) [reply app/assets/stylesheets/layout.scss]: Either use inline styling or a class instead

**Comment by MrBartusek** [comment]
#778 has been merged and #555 has been closed so this PR should be closed too

**Pull request closed without merging**